lib/services/presets_test.go

```diff
@@ -32,20 +32,29 @@ func TestAddRoleDefaults(t *testing.T) {
 	noChange := func(t require.TestingT, err error, i ...interface{}) {
 		require.ErrorIs(t, err, trace.AlreadyExists("no change"))
 	}
+	notModifying := func(t require.TestingT, err error, i ...interface{}) {
+		require.ErrorIs(t, err, trace.AlreadyExists("not modifying user created role"))
+	}
 
 	tests := []struct {
 		name                   string
 		role                   types.Role
 		enterprise             bool
-		reviewRequestsNotEmpty bool
-		accessReviewNotEmpty   bool
+		reviewNotEmpty         bool
+		accessRequestsNotEmpty bool
 
 		expectedErr require.ErrorAssertionFunc
 		expected    types.Role
 	}{
 		{
-			name:        "nothing added",
-			role:        &types.RoleV6{},
+			name: "nothing added",
+			role: &types.RoleV6{
+				Metadata: types.Metadata{
+					Labels: map[string]string{
+						types.TeleportManagedLabel: types.IsManaged,
+					},
+				},
+			},
```

Comment:
I don't think that we should assert that the labels change when we return an error and expect the role not to be written. Maybe this should actually trigger a change to write the new label

Reply:
No I misread, I thought this was `expected:`